api/v1beta1/hostedcluster_types.go

```diff
@@ -1110,6 +1110,10 @@ type AWSRolesRef struct {
 	//  "Statement": [
 	//    {
 	//      "Action": [
+	//        "autoscaling:DescribeAutoScalingGroups",
```

Comment:
Just for the record, we don't use autoscaling groups, so this should be transparent for us.

Reply:
Agree, this was added to reflect the list of permissions in AWS documentation: 
https://cloud-provider-aws.sigs.k8s.io/prerequisites/#iam-policies